test/e2e/node/pods.go

```diff
@@ -117,8 +117,11 @@ var _ = SIGDescribe("Pods Extended", func() {
 
 			ginkgo.By("verifying the kubelet observed the termination notice")
 
+			// allow up to 3x grace period (which allows process termination)
+			// for the kubelet to remove from api.  need to follow-up on if this
```

Comment:
Nit: Worth adding a brief description, either here or in the commit message, around how we arrived at 3 as the multiplier?